{"cve_repaire_en": "", "cve_code_exp_url": "", "cve_id": "CVE-2018-20718", "cve_categories": ["category1", "category2"], "cve_code_detect_url": "", "cve_name": "cve_name", "cve_risk_level": "determine", "cve_detail_en": "In Pydio before 8.2.2, an attack is possible via PHP Object Injection because a user is allowed to use the $phpserial$a:0:{} syntax to store a preference. An attacker either needs a \"public link\" of a file, or access to any unprivileged user account for creation of such a link.", "cve_name_en": "cve_name_en", "edit_time": "Sat Oct 12 14:43:51 2019", "cve_code_des_url": "", "cve_other_info": "", "modify_time": "2019-01-15", "cve_detail": "", "references": [{"URL": "https://blog.ripstech.com/2018/pydio-unauthenticated-remote-code-execution/", "Description": "MISC:https://blog.ripstech.com/2018/pydio-unauthenticated-remote-code-execution/"}], "cve_repaire": "", "editor": "warcup", "add_time": "2019-01-15"}